03-task-manager/starter/QuizAnswers2.txt

```diff
@@ -0,0 +1,11 @@
+In this lesson, you created a middleware function called asyncWrapper. Why? : To handle the try catch block of the async code we passed as a function parameter to it.
```

Comment:
Technically correct, but why make the wrapper function when we can just handle the try catch block in the controller?

Reply:
For a cleaner code..to move error handling in a separate file. Another reason I think is for reusability ( so that the same try catch block doesn't need to be typed for all api calls)